src/Models/MenuItem.php

```diff
@@ -22,6 +22,7 @@ class MenuItem extends Model
     const TYPE_INTERNAL_LINK = 1;
     const TYPE_EXTERNAL_LINK = 2;
     const TYPE_HTML = 3;
+    const TYPE_EMPTY = 4;
```

Comment:
What about the  `public function linkType(): ?int` method ? We can't detect if the type of menu item is `empty` right ?